Vic2ToHoI4/Source/V2World/Army.cpp

```diff
@@ -61,52 +39,53 @@ Vic2::Regiment::Regiment(std::istream& theStream)
 
 	if (type.empty())
 	{
-		LOG(LogLevel::Warning) << "Regiment or Ship " << name << " has no type";
+		Log(LogLevel::Warning) << "Regiment or Ship " << name << " has no type";
 	}
 }
 
+
 bool Vic2::Regiment::isMobilised() const
 {
-        auto* pop = Vic2::Pop::getByID(pop_id);
-        if (pop == nullptr)
-        {
-                return false;
-        }
-        if (pop->getType() == "soldiers")
-        {
-                return false;
-        }
-        return true;
+	auto* pop = Vic2::Pop::getByID(pop_id);
```

Comment:
I get jitters seeing this ;)

Reply:
Likewise. Need to do a cleanup of Vic2World at some point.